ddprof-stresstest/build.gradle

```diff
@@ -23,6 +23,11 @@ sourceSets {
 }
 
 jmh {
+  def javaHome = System.getenv("JAVA_TEST_HOME")
```

Comment:
Should this not be at a higher level ? Other tests would want to benefit from this ?
The way I understand this is valid only for the stress test ?